chain-extensions/common/src/lib.rs

```diff
@@ -0,0 +1,277 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![allow(non_snake_case)]
+
+use scale_info::prelude::vec::Vec;
+use sp_core::{Decode, Encode, MaxEncodedLen};
+use sp_runtime::{codec, ArithmeticError, DispatchError, TokenError};
+
+pub use spacewalk_primitives::{Asset, CurrencyId};
```

Comment:
These are not used here. I guess the idea is to re-export them so they can be used by other modules. However, I saw that the `token` extension also directly imports `spacewalk-primitives` so I'd say we either remove the spacewalk-primitives here or change the import in the `token` extension module.